docs/source/reference/modding/texture-modding/convert-bump-mapped-mods.rst

```diff
@@ -237,9 +237,9 @@ Since we want the glory of normal mapping in our OpenMW setup, we will go with t
 #. OpenMW detects normal maps if they have the same name as the base diffuse texture, but with a *_n.dds* suffix. In this mod, the normal maps has a suffix of *_nm.dds*. Change all the files that ends with *_nm.dds* to instead end with *_n.dds*.
 #. Finally, `we are done`_!
 
-Since these models have one or two textures applied to them, the fix was not that time-consuming. 
-It gets worse when you have to fix a model that uses loads of textures. The principle is the same, 
-it just requires more manual work which is annoying and takes time.
+Since these models have one or two textures applied to them, the fix was not that time-consuming. The process continues to work for more complex models that use more textures, but looking through each category for texture effects and normal mapped textures rapidly becomes tedious. Luckily, NifSkope provides a feature to do the same automatically. 
+
+Rightclick in NifSkope to access the *Spells* dropdown menu, also available via the top bar, open the *Blocks* section, and `click on the option Remove by ID`_. You can then input the RegEx expression ``^NiTextureEffect`` (directing it to remove any block whose name starts with "NiTextureEffect") to automatically remove all texture effect blocks within the NIF. This also has the helpful side effect of listing `all the blocks within the NIF in the bottom section`_, allowing you to additionally root out any blocks referencing *_nm.dds* textures without having to painstakingly open each category.  
```

Comment:
I think something like this should read better:

In NifSkope, press the right mouse button to access <...>
hover over *Block* section and choose *Remove By Id* action

Reply:
If you can use Git on your system, you could make the commit, squash all the commits into one then force-push the new commit history into the PR branch. Or the maintainer who merges the PR does that using the relevant button (I hope so). Unfortunately GitHub web interface doesn't allow the contributor to do the same.

If you're unable to use Git, by all means just push another commit.

Reply:
Perhaps leave a note for psi29a to squash the commits on merge.